examples/viz/tree_visualization.cpp

```diff
@@ -20,7 +20,51 @@
 #include <fstream>
 #include <random>
 
-#include <point_clouds.hpp>
+// FIXME: this is a temporary place for loadPointCloud and writePointCloud
+// Right now, only loadPointCloud is being used, and only in this example
+template <typename DeviceType>
+void loadPointCloud(std::string const &filename,
```

Comment:
What was the reason for moving this out of `point_clouds.hpp` to here?

Reply:
For examples/ to be self-contained, and work without benchmarks.